frontend/src/components/navbarDropDown/NavbarDropDown.tsx

```diff
@@ -14,6 +14,23 @@ import { Link, useNavigate } from "react-router-dom";
 import { Modal } from "components";
 import { ProfileModal } from "components/Modal/ProfileModal/ProfileModal";
 
+function useOutsideAlerter(
+    ref: any,
+    setActiveDropdownButton: React.Dispatch<React.SetStateAction<boolean>>
+) {
+    useEffect(() => {
+        function handleClickOutside(event: Event) {
+            if (ref.current && !ref.current.contains(event.target)) {
+                setActiveDropdownButton(false);
+            }
+        }
+        document.addEventListener("mousedown", handleClickOutside);
+        return () => {
+            document.removeEventListener("mousedown", handleClickOutside);
+        };
+    }, [ref]);
+}
+
```

Comment:
It is named useOutsideAlerter which indicates it is a hook. So it should be moved to specific folder "hooks"